modal/functions.py

```diff
@@ -1347,6 +1355,7 @@ def __call__(self, *args, **kwargs) -> Any:  # TODO: Generics/TypeVars
                 " a Modal container in the cloud",
             )
 
+    @synchronizer.no_input_translation
```

Comment:
Esp. important here to *only* skip input translations and not output, since the return value from `spawn()` *should* be synchronicity-wrapped.